tier4/{{cookiecutter.project_slug}}/.github/cookiecutter.json

```diff
@@ -17,6 +17,8 @@
     "software_type":["standalone/mobile", "standalone/iot", "standalone/desktop", "standalone/web", "standalone/backend", "standalone/other", "addon", "library", "configurationFiles"],
     "languages": "None",
     "repo_type" : ["Package", "Website", "Standards", "Libraries", "Data", "Apps", "Tools", "APIs", "Docs"],
+    "usage_type" : ["Open Source", "Government Wide Reuse", "Exempt By Law", "Exempt By National Security", "Exempt By Agency System", "Exempt By Agency Mission", "Exempt By CIO", "Exempt By Policy Date"],
+    "exemption" : "None",
```

Comment:
As per the schema, `exemptionText` is required if the `usageType` selected is an exemption value (exemptByLaw, exemptyByCIO, etc.) 

Is there a way to add this logic in `post_gen_hook.py` or in `cookiecutter.json` where we ask the user to fill out the `exemptionText` only if they have answered with an exemption value for `usageType`? Not sure if cookiecutter can support this conditionals hmmmmm

Reply:
i attempted to do this within the JSON using Jinja2 resources but I couldnt get to a solution. ended up putting the logic within post_gen hook so it asks the to enter an exemption text if they choose an exemption AND the user is finished filling out the cookiecutter